.travis.yml

```diff
@@ -1,7 +1,13 @@
 services:
   - mongodb
+  - postgresql
 language: go
 install: false
 go:
   - "1.10.x"
 script: go test ./...
+before_script:
+  - export POP_PATH=$PWD
+  - go get github.com/gobuffalo/pop/soda
+  - soda create -e development_postgres
+  - soda migrate up -e development_postgres
```

Comment:
you can replace these two lines with `buffalo db migrate up` if you have `buffalo` in the environment